docs/releasehistory.rst

```diff
@@ -7,6 +7,78 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
 * ``minor`` increments add features but do not break API compatibility
 * ``micro`` increments represent bugfix releases or improvements in documentation
 
+
+
+0.4.0 - Performance optimizations and support for SMIRNOFF 0.3 specification
+----------------------------------------------------------------------------
+
+This update improves runtime when running ``create_openmm_system`` on large topologies.
```

Comment:
Unclear what is meant by "improves runtime". Perhaps: 

"This update contains performance enhancements that significantly reduce the time to create OpenMM systems for topologies containing many molecules" via ``ForceField.create_openmm_system()``?

Reply:
Looks good. Added as written.